examples/private/main.tf

```diff
@@ -0,0 +1,224 @@
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "RPC" = {
+      description     = "Allow security-group only to access Redpanda RPC endpoint for intra-cluster communication"
+      from_port       = 33145
+      to_port         = 33145
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "Admin" = {
+      description     = "Allow anywhere inbound to access Redpanda Admin endpoint"
+      from_port       = 9644
+      to_port         = 9644
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "Grafana" = {
+      description     = "Allow anywhere inbound to access grafana end point for monitoring"
+      from_port       = 3000
+      to_port         = 3000
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "JavaOMB" = {
+      description     = "Allow anywhere inbound to access for Open Messaging Benchmark"
+      from_port       = 8080
+      to_port         = 8080
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "Prometheus" = {
+      description     = "Allow anywhere inbound to access Prometheus end point for monitoring"
+      from_port       = 9090
+      to_port         = 9090
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "NodeExporter" = {
+      description     = "node_exporter access within the security-group for ansible"
+      from_port       = 9100
+      to_port         = 9100
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+    "SchemaRegistry" = {
+      description     = "schema_registry access for external users"
+      from_port       = 8081
+      to_port         = 8081
+      protocol        = "tcp"
+      enabled         = true
+      cidr_blocks     = null
+      self            = true
+      security_groups = [aws_security_group.client_sec_group.id]
+    }
+  }
+}
+
+variable "public_key_path" {
+  type    = string
+  default = "~/.ssh/id_rsa.pub"
+}
+
+variable "nodes" {
+  type    = number
+  default = 3
+}
+
+variable "deployment_prefix" {
+  type    = string
+  default = "test-rp-cluster"
```

Comment:
Why test? Isn't this prod-ready usable for others? (I know they can change it, but just asking about the default)

Reply:
mostly so i could have a non-colliding static prefix. i can change it 🤷‍♀️ 

I would say this is productionish. I wouldn't recommend it as there's a lot more stuff you could do to make it more secure and robust, but you could get away with it if you needed to do so.